srv/components/system/network/files/dns_check.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+
+set -eu
+
+domain="${1:-$(hostname)}"
+
+echo "Checking DNS resolution for domain '$domain'"
+
+ns_servers=$(dig +short NS "$domain" | paste -sd " " -)
```

Comment:
Should we check if dig is installed?

Reply:
bind-utils provides it and we do install them [here](https://github.com/Seagate/cortx-prvsnr/blob/72d5c1e2a8375de6f8548791a4123770ac17bba8/srv/components/system/install.sls#L23)

Reply:
resolve that as `getent` now is used instead